src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -528,22 +528,4 @@ protected function getErrorMessageForException(Exception $e, array $messages)
             $e->getMessage()
         );
     }
-
-    /**
-     * @param Request $request
-     *
-     * @return Filters
-     */
-    protected function buildFiltersParamsByRequest(Request $request, Filters $filters)
```

Comment:
BC break 🔴  😱 

Even if it's now useless, we cannot remove it as there might be modules who have controllers extending FrameworkBundleAdminController and using this.

However we can deprecate it if it's usless now